README.md

```diff
@@ -12,30 +19,33 @@ of sudo privlages. Make sure that you can ssh as service to the target without a
 `ssh-copy-id service@<target hostname>`
 
 ## Installation
+**[TODO: Verify consistency of these instructions with the deployments implemented
+in the current "derived" applications.]**
 
-**1.)** First make sure a base set of applications are installed target host machine. Run `os_scripts/web_gui_support.sh`.
+**1.)** First make sure a base set of applications are installed target host machine or docker image - the
+equivalent of running `os_scripts/web_gui_support.sh`.
 
-**2.)** Install the `simple UI server`  
-- In `phpStorm`, open the project `simpleui/simpleui-server/`.
+**2.)** Build and install the `simple UI server`  
+- In a `JetBrains IDE` (`clion`, `webstorm`, or `phpStorm`), open the project `simpleui/simpleui-server/`.
 - Open the `package.json` file, then follow the prompts to run `npm install`.
 - Right-click `package.json`, and click `Show npm Scripts`.
 - In the `npm` window, click `build`.
 - In the `npm` window, click `copy-dist-to-output`.
 
-**3.)** Install the `base_app`  
-- In `phpStorm`, open the project `simpleui/base_app/`.
+**3.)** Build and install the `base_app`  
+- In the `JetBrains IDE`, open the project `simpleui/base_app/`.
 - Open `base_app/package.json`, then follow the prompts to run `npm install`.
 - Right-click `package.json`, and click `Show npm Scripts`.
 
 
 **3a)** **Production Build:**  
-- In the phpstorm `npm` window, click `build-client-prod` to produce a `dist.tgz` file.
-- In the phpstorm `npm` window, click `copy-dist-to-output` to copy the `dist.tgz` file to `../../../output/simpleui/base_app` for consumption by the deploy script.
+- In the JetBrains IDE's `npm` window, click `build-client-prod` to produce a `dist.tgz` file.
+- In the JetBrains IDE's `npm` window, click `copy-dist-to-output` to copy the `dist.tgz` file to `../../../output/simpleui/base_app` for consumption by the deploy script.
 
 **3b)** **Development Build:**  
 - Right-click `base_app/webpack.config.js` and follow the prompts to run it.
-- From the phpstorm `npm` window, select `build-client-dev` to produce a dist.tgz file.
-- In the phpstorm `npm` window, click `copy-dist-to-output` to copy the `dist.tgz` file to `../../../output/simpleui/base_app` for consumption by the deploy script.
+- From the JetBrains IDE's `npm` window, select `build-client-dev` to produce a dist.tgz file.
+- In the JetBrains IDE's `npm` window, click `copy-dist-to-output` to copy the `dist.tgz` file to `../../../output/simpleui/base_app` for consumption by the deploy script.
```

Comment:
My only comment on this doc is the use (requirement?)  of Jetbrains. That works for our purposes but if this is staying open source these directions should be using npm commands in the command line. Just a note for the future.